sale_quick_payment/__openerp__.py

```diff
@@ -22,7 +22,7 @@
 
 {
     'name': 'Sale Quick Payment',
-    'version': '0.2',
+    'version': '0.3',
     'category': 'Generic Modules/Others',
     'license': 'AGPL-3',
     'description': """
```

Comment:
Please extract description to a README.rst file, and use OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst